src/app/task-displayer/task-displayer.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+import * as TaskJson from './task_sample_data.json';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-task-displayer',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './task-displayer.component.html',
+  styleUrl: './task-displayer.component.scss'
+})
+export class TaskDisplayerComponent {
+  TaskList = TaskJson;
+
+  DeleteTask(t:any){
+    const index = this.TaskList.tasks.indexOf(t);
```

Comment:
Would be worth to introduce id’s to tasks since this approach won’t work once you have data coming from DB